src/NzbDrone.Core/DecisionEngine/DownloadDecisionMaker.cs

```diff
@@ -120,6 +112,10 @@ private IEnumerable<DownloadDecision> GetDecisions(List<ReleaseInfo> reports, Se
                         else
                         {
                             _aggregationService.Augment(remoteEpisode);
+
+                            remoteEpisode.CustomFormats = _formatCalculator.ParseCustomFormat(remoteEpisode);
```

Comment:
This means we won't have CF unless we have a series, but the CFs calculated without a series/augmenting would be half baked.

Reply:
Would only be the original language logic that's screwy right? Yea hard decision if no CFs are better than incomplete CFs. 

This will mean that nothing shows for those releases that can't have a series parsed in manual search right? Or does it supplement series from the search criteria somewhere in this case? If so, it's probably fine like this then.

Reply:
Yea, sounds good.